src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -791,9 +767,129 @@ private static synchronized Object getFifoNextSpanDataRaw(
             "Unable to update historical data queue diagnostic tags!", e);
       }
     }
+  }
 
-    // Return data as raw object (public method stub will convert to expected type)
-    return queueDataMap != null ? (Object) queueDataMap : (Object) queueDataList;
+  /**
+   * Perform a rapid-catch up request and update tracking parameters. This request will attempt to
+   * find the start of data in an aggressive manner. If not data points are found, returns the end
+   * of the search timestamp. If data points are found, returns the timestamp of the first data
+   * point found. For proper behavior, Record data in UTC.
+   *
+   * @param startTimeTrackerMsLong the start time for historical data, in milliseconds since epoch
+   * @param includeTagGroupA if tag group A data should be included
+   * @param includeTagGroupB if tag group B data should be included
+   * @param includeTagGroupC if tag group C data should be included
+   * @param includeTagGroupD if tag group D data should be included
+   * @return timestamp in milliseconds since epoch
+   * @throws Exception for exceptions related to checking if data is exported in UTC format
+   * @throws IOException for errors closing exporter
+   */
+  private static long PerformRapidCatchUp(
```

Comment:
Method name violates Google Java Format. Must follow camelCase.

Should be `performRapidCatchUp`

Reply:
changed